web/WEB-INF/web.xml

```diff
@@ -242,458 +242,642 @@
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>characterImage</servlet-name>
         <url-pattern>/characterImage</url-pattern>
+        <url-pattern>/newberry-french/characterImage</url-pattern>
+        <url-pattern>/newberry-italian/characterImage</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>FileUpload</servlet-name>
         <url-pattern>/FileUpload</url-pattern>
+        <url-pattern>/newberry-french/FileUpload</url-pattern>
+        <url-pattern>/newberry-italian/FileUpload</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>proxy</servlet-name>
         <url-pattern>/proxy</url-pattern>
+        <url-pattern>/newberry-french/proxy</url-pattern>
+        <url-pattern>/newberry-italian/proxy</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>ProjectsServlet</servlet-name>
         <url-pattern>/projects</url-pattern>
+        <url-pattern>/newberry-french/projects</url-pattern>
+        <url-pattern>/newberry-italian/projects</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>ManuscriptsServlet</servlet-name>
         <url-pattern>/manuscripts</url-pattern>
+        <url-pattern>/newberry-french/manuscripts</url-pattern>
+        <url-pattern>/newberry-italian/manuscripts</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>LoginServlet</servlet-name>
         <url-pattern>/login</url-pattern>
+        <url-pattern>/newberry-french/login</url-pattern>
+        <url-pattern>/newberry-italian/login</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>createManuscript</servlet-name>
         <url-pattern>/createManuscript</url-pattern>
+        <url-pattern>/newberry-french/createManuscript</url-pattern>
+        <url-pattern>/newberry-italian/createManuscript</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>ProjectServlet</servlet-name>
         <url-pattern>/project/*</url-pattern>
+        <url-pattern>/newberry-french/project/*</url-pattern>
+        <url-pattern>/newberry-italian/project/*</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>ProjectServlet</servlet-name>
         <url-pattern>/manifest/*</url-pattern>
+        <url-pattern>/newberry-french/manifest/*</url-pattern>
+        <url-pattern>/newberry-italian/manifest/*</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>CanvasServlet</servlet-name>
         <url-pattern>/canvas/*</url-pattern>
+        <url-pattern>/newberry-french/canvas/*</url-pattern>
+        <url-pattern>/newberry-italian/canvas/*</url-pattern>
     </servlet-mapping>
     <servlet-mapping>
         <servlet-name>FolioReport</servlet-name>
         <url-pattern>/folioReport</url-pattern>
+        <url-pattern>/newberry-french/folioReport</url-pattern>
```

Comment:
These may be superfluous. I had added them when we were making separate sites within vanilla TPEN, but it doesn't break to have them.